package/tests/testthat/_snaps/development/tables-efficacy-rbmit01.md

```diff
@@ -13,24 +13,24 @@
           Relative Reduction (%)                                     5.7%      
           p-value (RBMI)                                            0.8932     
       5                                                                        
-        Adjusted Mean (SE)                   -4.253 (0.651)     -2.812 (0.645) 
-          95% CI                            (-5.539, -2.968)   (-4.087, -1.537)
-        Difference in Adjusted Means (SE)                       1.442 (0.921)  
-          95% CI                                               (-0.377, 3.261) 
-          Relative Reduction (%)                                    -33.9%     
-          p-value (RBMI)                                            0.1195     
+        Adjusted Mean (SE)                   -4.251 (0.647)     -2.805 (0.644) 
+          95% CI                            (-5.528, -2.974)   (-4.077, -1.533)
+        Difference in Adjusted Means (SE)                       1.446 (0.914)  
+          95% CI                                               (-0.358, 3.250) 
+          Relative Reduction (%)                                    -34.0%     
+          p-value (RBMI)                                            0.1155     
```

Comment:
numbers changing? this seems a bit odd

Reply:
yeah I need your help on this. These are the values reported in CI. Note that the changes are relatively small

Reply:
i still cant find why these number were changed. is there a way that we can track the package versions? the tlg-c renv snapshot should do, right?

Reply:
Unfortunately the workflow itself is not very verbose when it comes to displaying session info. The only information is coming from the "Run staged dependencies" step output: https://github.com/insightsengineering/tlg-catalog/actions/runs/9744147645/job/26889152743

The good news is that we are working on this so hopefully this will be enhanced soon.

Reply:
fantastic! thanks Pawel. I love github logs everything